Sample Data/CEF/ReadMe.md

```diff
@@ -1,2 +1,67 @@
 # About
-This folder tracks sample data of CEF format and can be pushed to Azure Log Analytics CommonEventFormat 
+This folder tracks sample data of CEF format and can be pushed to Azure Log Analytics CommonEventFormat.

```

Comment:
This should include steps to how to push the CEF sample file using the CEF setup (agent, etc.) and push into Log Analytics CommonEventFormat table